Header header
float64 omega_r
float64 omega_l